visual-dotnet/SauceLabs.Visual/SauceLabs.Visual.csproj

```diff
@@ -1,41 +1,40 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>SauceLabs.Visual</RootNamespace>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://github.com/saucelabs/visual-csharp-client/blob/main/LICENSE</PackageLicenseUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
isn't it a MIT ?

Reply:
I picked the one from `saucectl`. We can move to MIT without any issue for sure.